lib/widgets/about_zulip.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+class AboutZulipPage extends StatefulWidget {
+  const AboutZulipPage({super.key});
+
+  static Route<void> buildRoute(BuildContext context) {
+    return MaterialPageRoute(builder: (context) => const AboutZulipPage());
+  }
+
+  @override
+  State<AboutZulipPage> createState() => _AboutZulipPageState();
+}
+
+class _AboutZulipPageState extends State<AboutZulipPage> {
+  PackageInfo? _packageInfo;
+
+  @override
+  void initState() {
+    super.initState();
+    (() async {
+      final result = await PackageInfo.fromPlatform();
+      setState(() {
+        _packageInfo = result;
+      });
+    })();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text("About Zulip")),
+      body: SingleChildScrollView(
+        child: SafeArea(
+          minimum: const EdgeInsets.all(8), // ListView pads vertical
+          child: Center(
+            child: ConstrainedBox(
+              constraints: const BoxConstraints(maxWidth: 400),
+              child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
+                if (_packageInfo != null) ...[
+                  ListTile(title: const Text('App version'), subtitle: Text(_packageInfo!.version)),
+                ],
```

Comment:
Hmm, awkward that this info is async.

If we do find ourselves showing a frame or two where we don't have the version number yet, I think it'd be better to still show this list item and just have some kind of placeholder for the actual value, rather than leave out the whole list item. That should reduce the amount of visible jank.

So e.g.:
```suggestion
                ListTile(
                  title: const Text('App version'),
                  subtitle: Text(_packageInfo?.version ?? '(…)')),
```

Reply:
To simulate that situation, you can insert an `await Future.delayed(…)`:
```diff
       final result = await PackageInfo.fromPlatform();
+      await Future.delayed(Duration(milliseconds: 20));
       setState(() {
         _packageInfo = result;
```